docs/how_to/single_node.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
```

Comment:
lets include an autogenerated header in the generated files, 

e.g. in the `.sh`:
```
# Code generated by doctest. DO NOT EDIT.
# source: github.com/m3db/m3/docs/...
```

same thing for markdown with ```<!-- ... -->``` comments